internal/armada/processor/event_nats.go

```diff
@@ -2,24 +2,20 @@ package processor
 
 import (
 	"context"
-	"fmt"
 
 	log "github.com/sirupsen/logrus"
 
-	"github.com/G-Research/armada/internal/armada/repository"
 	"github.com/G-Research/armada/internal/armada/server"
-	"github.com/G-Research/armada/internal/common/eventstream"
 	"github.com/G-Research/armada/internal/common/logging"
 	"github.com/G-Research/armada/pkg/api"
 )
 
 type StreamEventStore struct {
-	stream             eventstream.EventStream
 	PulsarSubmitServer *server.PulsarSubmitServer
 }
 
-func NewEventStore(stream eventstream.EventStream) *StreamEventStore {
-	return &StreamEventStore{stream: stream}
+func NewEventStore() *StreamEventStore {
+	return &StreamEventStore{}
```

Comment:
This file should probably be renamed

Also in ReportEvents - we should assume pulsar is enabled

Reply:
done